app/templates/vendor/orders.html

```diff
@@ -105,7 +108,22 @@ <h2 class="ui header">
         {% endif %}
 
     </div>
+    <div class="ui modal">
+        <div class="header">You are about to <b class="action-type"></b> this order:</div>
+        <div class="content">
+            <div class="ui form">
+                  <div class="field">
+                    <label>Please write any comments to be sent to the customer <em>(optional)</em>:</label>
+                    <textarea rows="2" class="comment-container"></textarea>
+                  </div>
+            </div>
+        </div>
 
+        <div class="confirm-action">
+            <button class="negative ui button tiny compact cancel">Cancel</button>
```

Comment:
When you say 'highlighted' what exactly do you mean by that? Do you mean make the action buttons a specific color (green for accept, red for deny) and use grey for all cancel actions?

Right now on merchant side ordering, the following appears when a merchant is about to confirm an order: 
![merchant ordering](http://puu.sh/nlBuO/2010faf602.png)

1 commit ahead, in the popup for vendor confirming an order
![vendor confirm](http://puu.sh/nlBB0/f10a1c9e82.png)